helm/cluster-api-provider-cloud-director-app/Chart.yaml

```
@@ -0,0 +1,15 @@
+apiVersion: v2
+version: 0.0.1
+appVersion: 0.0.1
```

As we use `architect` during CI/CD let's use this placeholder to let architect fill out the generated version numbers

```suggestion
version: "[[ .Version ]]"
appVersion: "[[ .AppVersion ]]"
```